docs/releases/1.7.5.txt

```diff
@@ -9,4 +9,6 @@ Django 1.7.5 fixes several bugs in 1.7.4.
 Bugfixes
 ========
 
-* ...
+* Fixed a regression that made custom fields inheriting from
```

Comment:
that prevented custom fields inheriting from ManyToManyField from being recognized